### DISCUSSION TITLE
Send tracking rates of moving objects to PHD2, e.g. Comet C/2022 E3 (ZTF)

### DISCUSSION THREAD

**Discussion by kopi23** [Stellarium/stellarium#3020] [Q&A]
Hello all,

seems like that after cloudy skies for about three month now, there might be a chance to image comet C/2022 E3 (ZTF). *fingers crossed*
Judging from images on AstroBin, I decided to use a small apo with 288mm focal length.
I started Stellarium, chose the cam/scope/reducer combination I'm planning to use, set the field of view and fixed it to a star.
Then I jumped four hours ahead and saw that the comet has moved almost out of the fov :/
![anim](https://user-images.githubusercontent.com/37673792/214438565-9802be49-026f-4db9-b95e-477dba632868.gif)
So I searched for how to track comets...
As it turns out, PHD2 does have a feature called '[Comet Tracking](https://openphdguiding.org/man-dev/Tools.htm#:~:text=PHD2%20provides%20a%20Comet%20Tracking,comet%20tracking%20rate%20to%20PHD2)' 
Here you can either set a fixed rate, train PHD2 on the object to calculate the rate or get the rate from a planetarium application.
I searched and found this feature request [Comet display of RA and DEC rates #949](https://github.com/Stellarium/stellarium/issues/949) which was implemented just a few days after it was created. (Thank you!!)
And after activating the field 'Proper motion' in the information tab of the configuration dialog, I could see them for the comet.
Hourly motion: dɑ=-0°54'34" dδ=+0°08'19"

So after a long (and maybe unnecessary) introduction, my two questions.

First, PHD2 takes either pixels/hr or arcsec/hr for RA and Dec. Is dɑ for RA and dδ for Dec?

Second, when looking at the hourly motion values and scrolling through time, I noticed that the values change.
PHD2 apparently has an interface to get the tracking rates for the 'Tracking Comets' feature. Does Stellarium support this?

Thank you, kopi

**Comment by gzotti** [comment]
"Hourly motion: dɑ=-0°54'34" dδ=+0°08'19"" means a motion of -0°54'34" per hour in Right Ascension and +0°08'19" per hour in Declination. 

If PHD2 interacts with Stellarium via RemoteControl plugin, it could read all given information. But I don't know this program.

**Comment by kopi23** [comment]
Thanks gzotti, that was quick.

PHD2 is used in astrophotography for guiding to fix tracking/backlash issues during long exposures.
As far as I understand the docu of PHD2, other applications like Stellarium can connect via http and send the data, not the other way around :/
